WIP Survey 2021 results (#1648)
writethedocs/www: docs/surveys/salary-survey/2021.rst

.. raw:: html

   <div class="survey-results">

*****************************************
Write the Docs Salary Survey 2021 Results
*****************************************

.. contents::
   :local:
   :depth: 2
   :backlinks: none

Introduction
============

> The Write the Docs Salary Survey aims to gather data about salaries for documentarians across the world, to help our community members determine what appropriate salary ranges are, and to provide a benchmark for future negotiations.

Author: ```suggestion
The Write the Docs Salary Survey gathers salary data for documentarians across the world, to help our community members determine what appropriate salary ranges are, and to provide a benchmark for future negotiations.
```

Here we present the results of the third annual survey, which gathered submissions between October 11 and December 20, 2021. Although employees and independent contractors filled out different survey forms this time around, all questions other than for Section 3 (Salary Information) were identical and the data from both forms was combined to produce this report. 

In total, 959 documentarians (887 employees and 72 independent contractors) filled out the 2021 survey – a 19% increase from 2020 and a 48% increase from 2019. 

.. raw:: html

   <figure>
      <object role="img" aria-label="Survey growth (2019-2021)" aria-describedby="figure-introduction_desc" type="image/svg+xml" data="/_images/2021-survey-growth.svg">
         <p id="figure-introduction_desc">Bar chart showing survey growth in number of respondents between 2019 and 2021</p>
      </object> 
      <figcaption>Figure: Survey growth (2019-2021)</figcaption>
   </figure>

.. figure:: images/2021/2021-survey-growth.svg
   :class: hide

Feedback
--------

> We’d love your thoughts on this survey, so that we can continue to refine it over the years. You can email us at support@writethedocs.org with your ideas.

Author: ```suggestion
We’d love your thoughts on this survey, so that we can continue to refine it. Email us at support@writethedocs.org with your ideas.
```

- how focused their role was on tasks related to documentation.

1.1 Basis of employment
-----------------------

.. container:: question
   
   .. rubric:: What we asked

   1. What is the basis of your employment?

      - I am an employee
      - I was an employee, but am not currently employed
      - I am an independent contractor, freelance operator, or self-employed
      - I was an independent contractor, freelance operator, or self-employed, but do not currently have work

Of the 959 respondents who completed the survey in 2021, 887 were employees (92.5% of the total) and 72 were independent contractors (7.5% of the total). 

.. raw:: html

   <figure>
      <object role="img" aria-label="Basis of employment (2021)" aria-describedby="figure-basis1_desc" type="image/svg+xml" data="/_images/2021-basis-of-employment.svg">
         <p id="figure-basis1_desc">Donut chart showing proportion of employee to contractor respondents</p>
      </object> 
      <figcaption>Figure: Basis of employment (2021)</figcaption>
   </figure>

.. figure:: images/2021/2021-basis-of-employment.svg
   :class: hide

> Although employees have always made up the majority of all respondents, there has been a slight increase in the proportion of independent contractors each year - from 5.7% in 2019, to 6.8% in 2020, to 7.5% in 2021. 

Author: ```suggestion
Although employees have historically made up the majority of all respondents, there has been a slight increase in the proportion of independent contractors each year - from 5.7% in 2019, to 6.8% in 2020, to 7.5% in 2021. 
```

Author: You talk about precentages, but show totals in the following graph, is that intentional>?

   | The expectations on me are realistic/reasonable             | 61.1% |
   +-------------------------------------------------------------+-------+
   | I'm compensated fairly for the work I do                    | 59.7% |
   +-------------------------------------------------------------+-------+
   | My work is sufficiently interesting and/or challenging      | 59.7% |
   +-------------------------------------------------------------+-------+
   | My contributions are valued                                 | 59.7% |
   +-------------------------------------------------------------+-------+
   | I feel respected                                            | 56.9% |
   +-------------------------------------------------------------+-------+
   | I have opportunities for career development and advancement | 18.1% |
   +-------------------------------------------------------------+-------+
   | I'm satisfied with my benefits                              | 15.3% |
   +-------------------------------------------------------------+-------+

Only 1 contractor chose not to select any positive aspects. 

Section 1: Organization demographics

> ====================================

Author: Section 4 I guess? 
You shouldn't have to number these yourself.

Author: Probably we tried this before and actually you do. Which makes me really sad.
